src/Idris/Parser.idr

```diff
@@ -390,7 +390,7 @@ mutual
            pure (PDotted (boundToFC fname b) b.val)
     <|> do b <- bounds (symbol "`(" *> expr pdef fname indents <* symbol ")")
            pure (PQuote (boundToFC fname b) b.val)
-    <|> do b <- bounds (symbol "`{{" *> name <* symbol "}" <* symbol "}")
+    <|> do b <- bounds (symbol "`{{" *> name <* symbol "}}")
```

Comment:
This was a bug fix cf. #838. Is this reintroducing the bug?

Reply:
This PR has a better solution to #838 and without [this problem](https://github.com/idris-lang/Idris2/pull/838#issuecomment-743215738),

Reply:
Nice! <3